src/ray/common/ray_config_def.h

```diff
@@ -397,6 +397,7 @@ RAY_CONFIG(int64_t, max_fused_object_count, 2000)
 RAY_CONFIG(bool, automatic_object_deletion_enabled, true)
 
 /// Grace period until we throw the OOM error to the application in seconds.
+/// In unlimited allocation mode, this is the time delay prior to fallback allocating.
```

Comment:
Maybe we should lower this by default once we have unlimited allocation enabled and have a default minimum otherwise.